adapter/operations.go

```diff
@@ -32,13 +40,6 @@ type OperationRequest struct {
 	OperationID       string
 }
 
-type Operation struct {
-	Type       int32             `json:"type,string,omitempty"`
-	Properties map[string]string `json:"properties,omitempty"`
```

Comment:
Please kindly give us more detail that why we split the Properties to other many attributes of the Operation's struct.

Reply:
Good question, I wondered about the usage of fields vs a properties' map as well, and I think we discussed it briefly earlier. 
Personally, I would have defined properties that are common to all operations and mandatory as fields, and a general properties map that can used to add arbitrary properties needed and of interest for a specific adapter. 
In this case properties that are used in the API would be fields, the rest properties in a map. For convenience constants for common keys can be provided. 
How are adapters supposed to extend `Operation`?

Reply:
@kumarabd not quite sure what :eyes: means...

Reply:
@Aisuko  you're right. Splitting it to properties doesn't make sense. Im changing it right away.